src/main/java/org/jax/mvar/utility/insert/InsertUtils.java

```diff
@@ -202,4 +200,45 @@ private static int getMvarStrainId(Connection connection, String strainName) thr
         }
         return mvarStrainId;
     }
+
+    /**
+     * Given a table name and a condition (can be null), return the count from a Select count(*)
+     * @param connection JDBC connection
+     * @param tableName table name
+     * @param mySqlCondition Query condition after "WHERE"
+     * @param stopId end of record id
+     * @return number of rows in table
+     */
+    public static int countFromTable(Connection connection, String tableName, String mySqlCondition, int stopId) throws SQLException {
+        int numberOfRecords = 0;
+        if (stopId == -1) {
+            // count all
+            PreparedStatement countStmt = null;
+            ResultSet resultCount = null;
+            String query;
+            try {
+                if (mySqlCondition != null)
+                    query = "SELECT COUNT(*) from " + tableName + " WHERE "+ mySqlCondition + ";";
```

Comment:
Probably low risk based on users, but I would generally advice voiding an open/flexible where clause. Also, it seems that this specific case is not used in the modification for this PR